### PR TITLE
Add a description to the README.md

### PR DESCRIPTION
**Pull request by MaroonShaded**
I've also edited some steps.
Intellij says it cleaned up imports so yeah, I guess that too

### CODE REVIEW

File: src/main/java/io/github/communitymod/CommunityMod.java [hunk 1]
```diff
@@ -1,12 +1,7 @@
 package io.github.communitymod;
 
 import io.github.communitymod.core.config.Config;
-import io.github.communitymod.core.init.BlockEntityInit;
-import io.github.communitymod.core.init.BlockInit;
-import io.github.communitymod.core.init.EntityInit;
-import io.github.communitymod.core.init.ItemInit;
-import io.github.communitymod.core.init.SoundsInit;
-import io.github.communitymod.core.init.StructureInit;
+import io.github.communitymod.core.init.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/CommunityMod.java:4]: Don't do this. Wildcard imports should be avoided.

File: src/main/java/io/github/communitymod/common/entities/BeanEntity.java [hunk 2]
```diff
@@ -11,14 +11,7 @@
...
 import net.minecraft.world.entity.ai.attributes.Attributes;
-import net.minecraft.world.entity.ai.goal.BreedGoal;
-import net.minecraft.world.entity.ai.goal.FloatGoal;
-import net.minecraft.world.entity.ai.goal.FollowParentGoal;
-import net.minecraft.world.entity.ai.goal.LookAtPlayerGoal;
-import net.minecraft.world.entity.ai.goal.PanicGoal;
-import net.minecraft.world.entity.ai.goal.RandomLookAroundGoal;
-import net.minecraft.world.entity.ai.goal.TemptGoal;
-import net.minecraft.world.entity.ai.goal.WaterAvoidingRandomStrollGoal;
+import net.minecraft.world.entity.ai.goal.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/common/entities/BeanEntity.java:14]: Don't do this. Wildcard imports should be avoided.

File: src/main/java/io/github/communitymod/common/entities/GooseEntity.java [hunk 3]
```diff
@@ -14,14 +14,7 @@
...
 import net.minecraft.world.entity.ai.control.MoveControl;
-import net.minecraft.world.entity.ai.goal.BreedGoal;
-import net.minecraft.world.entity.ai.goal.FloatGoal;
-import net.minecraft.world.entity.ai.goal.FollowParentGoal;
-import net.minecraft.world.entity.ai.goal.LookAtPlayerGoal;
-import net.minecraft.world.entity.ai.goal.PanicGoal;
-import net.minecraft.world.entity.ai.goal.RandomLookAroundGoal;
-import net.minecraft.world.entity.ai.goal.TemptGoal;
-import net.minecraft.world.entity.ai.goal.WaterAvoidingRandomStrollGoal;
+import net.minecraft.world.entity.ai.goal.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/common/entities/GooseEntity.java:17]: Don't do this. Wildcard imports should be avoided.

File: src/main/java/io/github/communitymod/core/init/ItemInit.java [hunk 4]
```diff
@@ -1,27 +1,14 @@
...
 import net.minecraft.world.entity.EquipmentSlot;
 import net.minecraft.world.food.FoodProperties;
-import net.minecraft.world.item.ArmorItem;
-import net.minecraft.world.item.BlockItem;
-import net.minecraft.world.item.BowlFoodItem;
-import net.minecraft.world.item.Item;
-import net.minecraft.world.item.Rarity;
-import net.minecraft.world.item.RecordItem;
-import net.minecraft.world.item.SwordItem;
+import net.minecraft.world.item.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/ItemInit.java:11]: Don't do this. Wildcard imports should be avoided.

File: src/main/java/io/github/communitymod/core/init/ItemInit.java [hunk 5]
```diff
@@ -1,27 +1,14 @@
...
-import java.util.HashSet;
-import java.util.Set;
-
 import io.github.communitymod.CommunityMod;
-import io.github.communitymod.common.items.BeanBelt;
-import io.github.communitymod.common.items.MiguelItem;
-import io.github.communitymod.common.items.OrbOfInsanity;
-import io.github.communitymod.common.items.SpecialItem;
-import io.github.communitymod.common.items.WhatSign;
+import io.github.communitymod.common.items.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/init/ItemInit.java:4]: Don't do this. Wildcard imports should be avoided.

File: src/main/java/io/github/communitymod/core/util/SpawnEggData.java [hunk 6]
```diff
@@ -12,6 +10,8 @@
 import net.minecraft.world.item.SpawnEggItem;
 import net.minecraft.world.level.block.DispenserBlock;
 
+import java.awt.*;
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/communitymod/core/util/SpawnEggData.java:13]: Don't do this. Wildcard imports should be avoided.

**Review by DaRealTurtyWurty — CHANGES_REQUESTED** [review]

**Comment by MaroonShaded** [comment]
I know, I don't like them either, but intellij. aaaa I'll try and remove

**Comment by MetalTurtle18** [comment]
> I know, I don't like them either, but intellij. aaaa I'll try and remove

Preferences>editor>code style>java>imports>Class count to import with '*' set to some high number.

**Review by DaRealTurtyWurty — APPROVED** [review]

**Pull request merged**